framework/util/CMakeLists.txt

```diff
@@ -98,6 +99,11 @@ target_link_libraries(gfxrecon_util platform_specific vulkan_registry ${CMAKE_DL
 if (MSVC)
     # These files may fail to compile with VS2017 and older, requiring the default section limit of 2^16 to be increased.
     set_source_files_properties(${CMAKE_SOURCE_DIR}/framework/generated/generated_vulkan_struct_to_string.cpp PROPERTIES COMPILE_FLAGS /bigobj)
+    if(NOT CMAKE_CL_64)
```

Comment:
It's not clear from context why `CMAKE_CL_64` affects the underscore in the linker option... a comment for this wizardry is sure to be appreciated later.

Reply:
Yes, good recommendation.  I've added some clarifying comments.  Verify change has been made and resolve if satisfied, please.